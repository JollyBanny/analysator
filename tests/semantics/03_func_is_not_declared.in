function b(var n: integer): integer;
begin
end;

begin
    c();
end.