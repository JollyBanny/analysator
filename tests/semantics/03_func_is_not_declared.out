[Line: 6; Column: 5]	procedure c is not declared
